feat: new about modal

Clusters opened the pull request:
![Screenshot 2025-02-16 180705](https://github.com/user-attachments/assets/d37d6d85-01bd-46e5-9e71-7566827ac1ae)
![Screenshot 2025-02-16 181030](https://github.com/user-attachments/assets/0dcf20a3-fb49-4295-861e-acd830527a4b)
![Image](https://github.com/user-attachments/assets/bfcf1a34-5085-4b3a-a76a-1911f6c253e2)

### Changes made

linked Watcharr version number in FaceMenu to releases page
added modal with about info to FaceMenu bottom
added 4 new SVG icons
- Github -> Fontisto | MIT
- Website -> Fluent UI MDL2 | MIT
- TMDB -> https://www.themoviedb.org/about/logos-attribution | attribution
- IGDB -> Simple Icons | CC0 1.0 Universal

closes #802

Clusters committed:
FaceMenu: Add about modal

added modal with about info to FaceMenu bottom.
added 4 new SVG icons.

Clusters committed:
FaceMenu: npm format

IRHM commented on src/lib/nav/FaceMenu.svelte:
```diff
@@ -85,5 +92,93 @@
 	{/if}
-	<span>v{__WATCHARR_VERSION__}</span>
+	<span style="color:black">
+		<a
```
I think this would be better as a `<button>` so that when clicked we don't add the `#` to the url (then we don't mess with the back button in the browser tab).

IRHM commented on src/lib/nav/FaceMenu.svelte:
```diff
@@ -85,5 +92,93 @@
 	{/if}
-	<span>v{__WATCHARR_VERSION__}</span>
+	<span style="color:black">
+		<a
```
I think this (the about text) would be better as a `<button>` so that when clicked we don't add the # to the url (then we don't mess with the back button in the browser tab).

Clusters replied:
I see! Fixed.

IRHM commented on src/lib/nav/FaceMenu.svelte:
```diff
@@ -85,5 +92,93 @@
 </Menu>
+
+{#if aboutModalOpen}
+	<Modal title="" onClose={closeAbout} maxWidth="400px">
```
Because of the added styles below, I think it would be best if this modal was moved to its own file (AboutModal.svelte)

Clusters replied:
Seems to be an useful approach. Fixed

IRHM commented on src/lib/nav/FaceMenu.svelte:
```diff
@@ -85,5 +92,93 @@
+{/if}
+
+<style lang="scss">
+	h2.norm {
```
Is this style unused?

Clusters replied:
You were right! I had tested this inside the browser and have missed to remove the norm class there. Good find! Fixed

IRHM commented on src/lib/nav/FaceMenu.svelte:
```diff
@@ -85,5 +92,93 @@
+				</h5>
+				<div class="horizontal-icon-list">
+					<a
+						href="https://github.com/sbondCo/Watcharr"
```
A quick improvement on the github/website links might be to add a tooltip, so on hover the user can tell what the links are for (we have a tooltip action already, usable like this: `use:tooltip={{ text: "Liked", pos: "bot" }}`).

Or we could add some text for the links if you think that would be better.

Clusters replied:
I tried to implement this as well, because consistency is gold.
But I had the issue that the tooltip seem to have issues with the element size. Has it a fixed offset?

![Screenshot 2025-02-17 182716](https://github.com/user-attachments/assets/c11f502f-f999-4753-8f1f-01568c5ded1d)
![Screenshot 2025-02-17 182915](https://github.com/user-attachments/assets/12bd6ad6-6051-4896-a9b7-3fcca5d25a65)

IRHM replied:
I have no idea why it's doing that, I thought I had it take into account the elements size, but see my new comment below I made a few changes.

IRHM reviewed (changes requested):
Awesome job @Clusters this is great! Thank you for getting this done.

I have added a few comments which I think may improve upon the code (mostly small stuff). I don't mind doing them myself if you'd like as they are small if you agree with the notes, let me know.

Thanks!

Clusters commented:
Thank you for the review and feedback I'll make sure to work them in 👍🏻

Clusters committed:
Merge branch 'sbondCo:dev' into dev

Clusters committed:
FaceMenu: refactor about modal

- moved about modal into its own file
- minor fixes

IRHM committed:
AboutModal: Add text to gh/site links, use Modal title prop, add attribution file link

, added a hover effect to all links in the modal and removed the margin on h5.

Also updated the top paragraph.

IRHM commented:
Hey @Clusters, thank you for updating the modal, it looks really great!

I made a few small changes to the modal in this commit: https://github.com/sbondCo/Watcharr/pull/811/commits/0d69751fb84a8b894867759813be1d58d033c414

Mainly:

- Added text to the github/website links instead of going the tooltip route.
- Updated the main paragraph of the modal (now just explains the app, I felt weird having my name link there so I decided to rewrite it :())
- Added a link to our attribution file (which should link to the correct version branch).

![image](https://github.com/user-attachments/assets/7cabe9a8-1a5c-44fc-95e0-100badb1ace9)

I don't want to force my changes on your branch (sorta did sorry), let me know what you think!

Clusters commented:
> Hey @Clusters, thank you for updating the modal, it looks really great!
> 
> I made a few small changes to the modal in this commit: [0d69751](https://github.com/sbondCo/Watcharr/commit/0d69751fb84a8b894867759813be1d58d033c414)
> 
> Mainly:
> 
>     * Added text to the github/website links instead of going the tooltip route.
> 
>     * Updated the main paragraph of the modal (now just explains the app, I felt weird having my name link there so I decided to rewrite it :())
> 
>     * Added a link to our attribution file (which should link to the correct version branch).
> 
> 
> ![image](https://private-user-images.githubusercontent.com/37304121/414879812-7cabe9a8-1a5c-44fc-95e0-100badb1ace9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDAwNjg3MzYsIm5iZiI6MTc0MDA2ODQzNiwicGF0aCI6Ii8zNzMwNDEyMS80MTQ4Nzk4MTItN2NhYmU5YTgtMWE1Yy00NGZjLTk1ZTAtMTAwYmFkYjFhY2U5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjIwVDE2MjAzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdmOWEwYTk0MTEzODUwYjg3NDZiOTE5MDNiMjNjYmQ1NjhkYmJjMjgyZjNkYzU5ZmRhNmJhMWYwMWVkZDcxM2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.20FO8wH5bBpcdS_gUs1Aa4RxNz4RzJILy6dDlWhbGEg)
> 
> I don't want to force my changes on your branch (sorta did sorry), let me know what you think!

looks good, I like it!

Just did I short check, everything checks out :)

Thanks!

IRHM committed:
AboutModal: Remove unused style

IRHM committed:
Menu: Use color variables

IRHM reviewed (approved):
Thank you for all your work put into this feature, I think it turned out great!
